src/SeleniumLibrary/keywords/tableelement.py

```diff
@@ -194,13 +194,13 @@ def table_should_contain(self, locator, expected, loglevel='TRACE'):
                                  % (locator, expected))
 
     def _find_by_content(self, table_locator, content):
-        return self._find(table_locator, '//*', content)
+        return self._find(table_locator, 'xpath:.//*', content)
 
     def _find_by_header(self, table_locator, content):
-        return self._find(table_locator, '//th', content)
+        return self._find(table_locator, 'xpath:.//th', content)
 
     def _find_by_footer(self, table_locator, content):
-        return self._find(table_locator, '//tfoot//td', content)
+        return self._find(table_locator, 'xpath:.//tfoot//td', content)
```

Comment:
If support for chained locators is added so that it can be used also externally, not only internally, `.//xxx` should be recognized as an xpath expression automatically.

Reply:
Good point and perhaps `.//xxx` should be recognized as xpath. Should be easy to add, but in separate PR.